drake/multibody/rigid_body_plant/visualization/contact_viz.py

```diff
@@ -0,0 +1,90 @@
+# note, this script runs in the main context of drake-visulizer
+# where many modules and variables already exist in the global scope
+
+from director import lcmUtils
+from director import applogic
+import numpy as np
+import drake as lcmdrakemsg
+
+
+class ContactVisualizer(object):
+    def __init__(self):
+        self.folder_name = 'Contact Results'
+        self.name = "Contact Visualizer"
+        self.enabled = True
+        # Subscribes to the LCM topic
+        lcmUtils.addSubscriber(
+            'CONTACT_RESULTS',
+            messageClass=lcmdrakemsg.lcmt_contact_results_for_viz,
+            callback=self.handle_message)
+
+    def is_enabled(self):
+        return self.enabled
+
+    def set_enabled(self, enable):
+        self.enabled = enable
+
+    def handle_message(self, msg):
+        # Removes the folder completely.
+        # om is a magic object from director
+        om.removeFromObjectModel(om.findObjectByName(self.folder_name))
```

Comment:
if you want to be more explicit about these module names, you can add these imports to your script:

```
from director import objectmodel as om
from director import visualization as vis
from director.debugVis import DebugData
```

Since the script runs in the main context, many of these happen to be imported already, but these would be necessary if your code was run as a module.